calendar/experiments/calendar/parent/ext-calendar-provider.js

```diff
@@ -355,7 +355,10 @@ this.calendar_provider = class extends ExtensionAPI {
             name: "calendar.provider.onItemRemoved",
             register: fire => {
               let listener = (event, calendar, item) => {
-                return fire.async(convertCalendar(context.extension, calendar), item);
+                return fire.async(
+                  convertCalendar(context.extension, calendar),
+                  convertItem(item, {}, context.extension),
```

Comment:
If you don't mind, could you add `extraParameters` like for `onItemUpdated` and pass that in as options?

Reply:
Done.